packaging/installer/kickstart.sh

```diff
@@ -1277,6 +1235,93 @@ is_netdata_running() {
   fi
 }
 
+write_claim_config() {
+  if [ -z "${INSTALL_PREFIX}" ] || [ "${INSTALL_PREFIX}" = "/" ]; then
+    config_path="/etc/netdata"
+    netdatacli="$(command -v netdatacli)"
+  elif [ "${INSTALL_PREFIX}" = "/opt/netdata" ]; then
+    config_path="/opt/netdata/etc/netdata"
+    netdatacli="/opt/netdata/bin/netdatacli"
+  elif [ ! -d "${INSTALL_PREFIX}/netdata" ]; then
+    config_path="${INSTALL_PREFIX}/etc/netdata"
+    netdatacli="${INSTALL_PREFIX}/usr/sbin/netdatacli"
+  else
+    config_path="${INSTALL_PREFIX}/netdata/etc/netdata"
+    netdatacli="${INSTALL_PREFIX}/netdata/usr/sbin/netdatacli"
+  fi
+
+  claim_config="${config_path}/claim.conf"
+
+  if [ "${DRY_RUN}" -eq 1 ]; then
+    progress "Would attempt to write claiming configuration to ${claim_config}"
+    return 0
+  fi
+
+  progress "Writing claiming configuration to ${claim_config}"
+
+  config="[global]"
+  config="${config}\n    url = ${NETDATA_CLAIM_URL}"
+  config="${config}\n    token = ${NETDATA_CLAIM_TOKEN}"
+  if [ -n "${NETDATA_CLAIM_ROOMS}" ]; then
+      config="${config}\n    rooms = ${NETDATA_CLAIM_ROOMS}"
+  fi
+  if [ -n "${NETDATA_CLAIM_PROXY}" ]; then
+      config="${config}\n    proxy = ${NETDATA_CLAIM_PROXY}"
+  fi
+  if [ -n "${NETDATA_CLAIM_INSECURE}" ]; then
+      config="${config}\n    insecure = ${NETDATA_CLAIM_INSECURE}"
+  fi
+
+  run_as_root touch "${claim_config}.tmp" || return 1
+  run_as_root chmod 0640 "${claim_config}.tmp" || return 1
+  run_as_root chown ":${NETDATA_CLAIM_GROUP}" "${claim_config}.tmp" || return 1
+  run_as_root echo "${config}" > "${claim_config}.tmp" || return 1
+  run_as_root mv -f "${claim_config}.tmp" "${claim_config}" || return 1
```

Comment:
- `"${NETDATA_CLAIM_GROUP}` is not set
- `claim.conf` will be root:root 640 (not readable by `netdata`)

Reply:
```suggestion
  run_as_root touch "${claim_config}.tmp" || return 1
  run_as_root chmod 0640 "${claim_config}.tmp" || return 1
  run_as_root chown ":${NETDATA_CLAIM_GROUP:-netdata}" "${claim_config}.tmp" || return 1
  run_as_root echo "${config}" > "${claim_config}.tmp" || return 1
  run_as_root mv -f "${claim_config}.tmp" "${claim_config}" || return 1
```